src/lib/tuning/index/column_ref.hpp

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <iostream>
+#include <string>
+#include <vector>
+
+#include "types.hpp"
+
+namespace opossum {
+
+/**
+ * A ColumnRef is used to reference a set of columns by a table name
+ * and their column indexes in this table.
+ * This is used by the IndexEvaluator, IndexChoices and IndexOperations.
+ */
+struct ColumnRef {
```

Comment:
Unfortunately, I can‘t look into the code base right now, but don‘t we already have `ColumnReferences` in the LQP?

Even if we don‘t, you are referencing a set of columns, not a column